browser/index.mjs

```diff
@@ -54,6 +47,34 @@ const tokenIssues = (token) => {
   )
 }
 
+const alwaysTrue = always(true)
+const or = (before, token) => (issue) => before(issue) || token(issue)
+const and = (before, token) => (issue) => before(issue) && token(issue)
+const tokenMatches = (token) => {
+  token = token.startsWith('/') ? `${token.slice(1)}/` : token
+  return (issue) => issue.textContent.includes(token)
+}
+
+const issueMatches = (tokens, predicate = alwaysTrue) => {
+  if (isEmpty(tokens)) return predicate
+
+  const [token, ...rest] = tokens
+
+  // Drop or when no token after or
+  if (token === 'or' && isEmpty(rest)) return predicate
+
+  // Drop or if followed by an or (drop empty or)
+  if (token === 'or' && rest[0] === token) return issueMatches(rest, predicate)
+
+  if (token === 'or') {
+    const [next, ...nextRest] = rest
+
+    return issueMatches(nextRest, or(predicate, tokenMatches(next)))
+  }
+
+  return issueMatches(rest, and(predicate, tokenMatches(token)))
+}
+
```

Comment:
Rewrote to a recursive function. Would be nice to pull to a module and write tests for. Should do later.

Reply:
Very nice